source/includes/api/_subscriptions.md

```diff
@@ -500,14 +501,14 @@ curl -X PATCH https://api.resourcewatch.org/v1/subscriptions/:id/send_confirmati
 
 ## Modify subscription
 
-To modify a subscription:
-
 ```shell
 curl -X PATCH https://api.resourcewatch.org/v1/subscriptions/:id \
 -H "Authorization: Bearer <your-token>"
 ```
 
-With a body with the same format as before.
+To modify a subscription, use the following PATCH endpoint. The body should be formatted in the same way as when creating a subscription.
+
+If the request comes from another micro service, then it is possible to modify subscriptions belonging to other users. Otherwise, you can only modify subscriptions if you are the owner of the subscription.
```

Comment:
same as before - not sure if this should be here or not

Reply:
Keeping this unresolved because it might need to be changed.

Reply:
Same here, moved this content into a new section in the developer docs and added a link here.